app/assets/stylesheets/application.css

```diff
@@ -12,4 +12,6 @@
  *
  *= require_tree .
  *= require_self
+ *= require foundation_and_overrides
```

Comment:
If you have issues in the future with getting custom css/scss files to "take," you may want to come back and load foundation_and_overrides first here in the css manifest. As it stands right now, the `foundation_and_overrides` file will load last and therefore take precedence over other styles declared in files also in this folder (which are loaded with 'require_tree .'